<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="HA1101901716" name="com.avaya.sdmclient.runnerdemo.SecondThreadNew" tests="8" failures="0" timestamp="9 Nov 2015 06:36:13 GMT" time="1158.273" errors="1">
  <testcase name="startNewThread" time="33.458" classname="com.avaya.sdmclient.runnerdemo.SecondThreadNew"/>
  <testcase name="RefreshVM" time="31.739" classname="com.avaya.sdmclient.runnerdemo.SecondThreadNew">
    <error type="com.avaya.sdmclient.extraResources.MyException" message="Check if button is enabled or not..">
      <![CDATA[com.avaya.sdmclient.extraResources.MyException: Check if button is enabled or not..
	at com.avaya.sdmclient.Settings.clickButtonxPath(Settings.java:1865)
	at com.avaya.sdmclient.runnerdemo.SecondThreadNew.RefreshVM(SecondThreadNew.java:254)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:648)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:834)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1142)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:124)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:771)
	at org.testng.TestRunner.run(TestRunner.java:621)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1176)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1172)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1101)
	at org.testng.TestNG.run(TestNG.java:1009)
	at com.avaya.sdmclient.runnerdemo.settingsForConcThreads.runThread(settingsForConcThreads.java:306)
	at com.avaya.sdmclient.runnerdemo.SecondThreadNew$1MyRunnable.run(SecondThreadNew.java:418)
	at java.lang.Thread.run(Unknown Source)
]]>
    </error>
  </testcase> <!-- RefreshVM -->
  <testcase name="DeleteVM" time="62.464" classname="com.avaya.sdmclient.runnerdemo.SecondThreadNew"/>
  <testcase name="AddVMSuite" time="563.713" classname="com.avaya.sdmclient.runnerdemo.SecondThreadNew"/>
  <testcase name="StartVM" time="156.580" classname="com.avaya.sdmclient.runnerdemo.SecondThreadNew"/>
  <testcase name="RestartVM" time="160.546" classname="com.avaya.sdmclient.runnerdemo.SecondThreadNew"/>
  <testcase name="EditVM" time="44.241" classname="com.avaya.sdmclient.runnerdemo.SecondThreadNew"/>
  <testcase name="StopVM" time="105.532" classname="com.avaya.sdmclient.runnerdemo.SecondThreadNew"/>
</testsuite> <!-- com.avaya.sdmclient.runnerdemo.SecondThreadNew -->
